common/koa-middleware/withToggles.js

```diff
@@ -33,16 +35,48 @@ const parseCookies = function(req) {
 function withToggles(ctx, next) {
   const cookies = parseCookies(ctx.req);
   const togglesCookies = cookies.filter(cookie =>
-    cookie.key.startsWith('toggle_')
+    cookie.key.startsWith(prefixName)
   );
   const toggles = togglesCookies.reduce((acc, cookie) => {
     return Object.assign({}, acc, {
-      [cookie.key.replace('toggle_', '')]: cookie.value === 'true',
+      [cookie.key.replace(prefixName, '')]: cookie.value === 'true',
     });
   }, {});
 
   ctx.toggles = { ...defaultToggleValues, ...toggles };
+  enableDisableToggler(ctx);
   return next();
 }
 
+function enableDisableToggler(ctx) {
+  const validToggle = (toggles, toggleItem) => {
```

Comment:
This can exist outside of the main function as it's own thing.

Reply:
We've given `toggleItem` / `togglesFeature` a couple different names.

Would it not just be easier just to have `toggles.includes(toggle)` in the body of the method?

Reply:
agreed.  much cleaner!